docs/reference/lambdas.md

```diff
@@ -70,20 +70,14 @@ fun <T, R> List<T>.map(transform: (T) -> R): List<R> {
 This function can be called as follows:
 
 ``` kotlin
-val doubled = ints.map {it -> it * 2}
+val doubled = ints.map { it -> it * 2 }
 ```
 
 One other helpful convention is that if a function literal has only one parameter,
 its declaration may be omitted (along with the `->`), and its name will be `it`:
 
 ``` kotlin
-ints map {it * 2} // Infix call + Implicit 'it'
-```
-
-These conventions allow to write [LINQ-style](http://msdn.microsoft.com/en-us/library/bb308959.aspx) code:
```

Comment:
Probably not very relevant, but I think it would be nice if something from the official documentation would be google-able with queries like "kotlin linq"